### ISSUE TITLE
File and Telnet/SSH modules with Java applets no longer work

### ISSUE THREAD

**Issue by niggemannmj** [webmin/webmin#516]
The Java File Manager and Telnet/SSH modules each utilize Java applets. These do not work practically anymore for a variety of browsers. Even with Internet Explorer 11, they won't work because the applets are not signed. 

Furthermore, in the near future as Java 9 becomes the public forward release for Oracle, there will no longer be a Java plug-in for even for browsers that still support them. Eventually the final Java 8 plug-in will expire and thus ends applets running in browsers.

The Java File Manager and Telnet/SSH modules should be retired.

**Comment by jcameron** [comment]
Yeah, they are on their way out in Webmin. The Java file manager has been replaced by the new HTML File Manager module, and we have an AJAX-based login module.

I'm reluctant to completely remove them yet though, as there are users who really like the Java file manager still!
